ingestjob/logger.go

```diff
@@ -0,0 +1,166 @@
+package ingestjob
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/LF-Engineering/insights-datasource-shared/uuid"
+	jsoniter "github.com/json-iterator/go"
+)
+
+const (
+	logIndex   = "insights-job-logging"
+	inProgress = "inprogress"
+	failed     = "failed"
+	done       = "done"
+)
+
+// ESLogProvider used in connecting to ES logging server
+type ESLogProvider interface {
+	CreateDocument(index, documentID string, body []byte) ([]byte, error)
```

Comment:
can we also write a function to generate this documentID? This function would be common for all datasource connectors.

Reply:
yep, lets add that as a part of another pull request.

Reply:
No need for having it in a separate func cuz it's used only within write func